cloudify_agent/installer/__init__.py

```diff
@@ -198,14 +198,14 @@ def _create_agent_env(self):
             # broker_ip might not be set yet, and retrieved from the manager
 
             env.CLOUDIFY_BROKER_PORT: self.cloudify_agent.get('broker_port'),
-            env.CLOUDIFY_FILE_SERVER_PORT:
-                self.cloudify_agent['file_server_port'],
-            env.CLOUDIFY_FILE_SERVER_PROTOCOL:
-                self.cloudify_agent.get('file_server_protocol'),
             env.CLOUDIFY_REST_PORT:
                 self.cloudify_agent.get('rest_port'),
             env.CLOUDIFY_REST_PROTOCOL: self.cloudify_agent.get(
                 'rest_protocol'),
+            env.CLOUDIFY_FILE_SERVER_PORT:
```

Comment:
since it's the same (in terms of being mandatory or not) as the rest_port and protocol